# Request: Only requesting one thing, so no need for a request variable

---
# Response: A vector of string commands for the supervisor to execute
bool home
int8 status
